mcy/src/components/Layout/Drawer.jsx

```diff
@@ -1,5 +1,117 @@
-const Drawer = () => {
-  return <div>Drawer</div>;
+/* eslint-disable no-unused-vars */
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { styled } from "@mui/material";
+import Stack from "@mui/material/Stack";
+import Box from "@mui/material/Box";
+import DrawerSlide from "@mui/material/Drawer";
+import Button from "@mui/material/Button";
+import List from "@mui/material/List";
+import ListItemText from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ClearIcon from "@mui/icons-material/Clear";
+import CheckIcon from "@mui/icons-material/Check";
+
+// eslint-disable-next-line react/prop-types
+const Drawer = ({ open, setOpen }) => {
+  const Navigate = useNavigate();
+
+  const toggleDrawer = (newOpen) => () => {
+    setOpen(newOpen);
+  };
+
+  const handleGoToNews = () => {
+    Navigate("/news");
+  };
+
+  const handleGoToAttendance = () => {
+    Navigate("/Attendance");
+  };
+
+  const DrawerList = (
+    <DrawerWapper rol="presentation">
+      <DrawerTop>
+        <ExitIcon onClick={toggleDrawer(false)} />
+      </DrawerTop>
+      <List>
+        <ListButton>
+          <ListIcon>
+            <ListCheckIcon />
+          </ListIcon>
+          <ListText onClick={handleGoToNews}>소식</ListText>
+        </ListButton>
+        <ListButton>
+          <ListIcon>
+            <ListCheckIcon />
+          </ListIcon>
+          <ListText onClick={handleGoToAttendance}>출석</ListText>
+        </ListButton>
+        <ListButton>
+          <ListIcon>
+            <ListCheckIcon />
+          </ListIcon>
+          <ListText
+            onClick={() => {
+              Navigate("/birthDay");
+            }}
+          >
+            생일
+          </ListText>
+        </ListButton>
+      </List>
+    </DrawerWapper>
+  );
+
+  return (
+    <>
+      <Button onClick={toggleDrawer(true)}></Button>
+      <DrawerSlide open={open} onClose={toggleDrawer(false)} anchor="right">
+        {DrawerList}
+      </DrawerSlide>
+    </>
+  );
 };
 
+const DrawerWapper = styled(Box)`
+  width: 213px;
```

Comment:
```suggestion
  width: 213px;
```
이건 가급적 퍼센트로 정의해주거나 별도로 정의해주지 않고 콘텐츠 내용에 따라 달라지게 하는게 나음